dc-infrastructure.tf

```diff
@@ -145,6 +147,9 @@ module "bitbucket" {
   db_instance_class       = var.bitbucket_db_instance_class
   db_iops                 = var.bitbucket_db_iops
 
+  local_home_size  = var.bitbucket_local_home_size
```

Comment:
We need to add local_home_size for other products

Reply:
🤦 thanks Jun.